packages/mui-base/src/MenuItemUnstyled/MenuItemUnstyled.tsx

```diff
@@ -105,6 +107,10 @@ MenuItemUnstyled.propTypes /* remove-proptypes */ = {
    * @default false
    */
   disabled: PropTypes.bool,
+  /**
+   * Used for keyboard text navigation matching.
```

Comment:
It's good to first describe what is it, then explain what it's used for:

```suggestion
   * A text representation of the menu item's content.
   * Used for keyboard text navigation matching.
```